Formula/cppad.rb

```diff
@@ -2,8 +2,8 @@ class Cppad < Formula
   desc "Differentiation of C++ Algorithms"
   homepage "https://www.coin-or.org/CppAD"
   # Stable versions have numbers of the form 201x0000.y
```

Comment:
Wondering if that comment is still necessary. Looks like we don't follow that rule anymore.

Reply:
let me run a quick check

Reply:
[it was introduced five years ago](https://github.com/Homebrew/homebrew-core/commit/c551a3b504700e9dd14c70ea00cae523b5cf75a6), I think it should be covered by the livecheck block now.